src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -204,9 +205,10 @@ def ok_to_stop(self, daemon_ids: List[str]) -> bool:
         })
 
         if ret:
-            logger.info(f'It is NOT safe to stop {names}: {err}')
+            logger.error(f'It is NOT safe to stop {names}: {err}')
             return False
```

Comment:
can we also return `err` here?

Reply:
I've added a commit to return an HandleCommandResult from the ok-to-stop command.